seed/urls/main.py

```diff
@@ -151,6 +153,43 @@
         name='delete_organization_buildings'
     ),
 
+    # timeseries data
```

Comment:
The feature as a whole.

As for the actual paths for the endpoints, I think the `_url` suffix should be removed.